co2birthdate/website
# Next.js based static website

**How to test**
```
npm run export
npm start
```
Open http://localhost:300 and verify the website.
Deploy to branch gh-pages
```
npm run deploy
```
Update the repository settings by mapping _gh-pages_ as a source branch of Github Pages. In a minute or two, check https://co2birth.date
